tests/Tests/Models/Enums/Library.php

```diff
@@ -11,8 +11,10 @@
 use Doctrine\ORM\Mapping\GeneratedValue;
 use Doctrine\ORM\Mapping\Id;
 use Doctrine\ORM\Mapping\OneToMany;
+use Doctrine\ORM\Mapping\Table;
 
 #[Entity]
+#[Table('library1')]
```

Comment:
Would the following fix the issue?

```suggestion
#[Table('`library`')]
```

[The docs](https://www.doctrine-project.org/projects/doctrine-orm/en/3.5/reference/basic-mapping.html#quoting-reserved-words) suggest it would.

Reply:
Yes, indeed.
I had seen backticks used in definitions, but I always thought it was because the underlying platform was MySQL, as double quotes are the SQL standard for quoting identifiers. Thanks!